File: FirstBitSolutions/C Module/Assignments/Assignment 4/Ques1.c
//1. Print armstrong numbers in the given range 1 to n.
#include<stdio.h>
void main()
{
	for(int k=1;k<=10000;k++){
		int no=k;
		int temp=no;
		int count=0;
		while(no>0){
			count++;
			no=no/10;
		}
		no=temp;
		int sum=0,rem;
		int res;
		while(no>0){
			rem=no%10;
			res =1;
			for(int i=1;i<=count;i++)
				
				res=res*rem;
			
				sum=sum+res;
				no=no/10;
			
		}
		if(sum==temp)
		printf("%d are armstrong numbers\n",temp);
	}

}

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 4/Ques1.java
class Ques{ 
public static void main(String args[]){ 
for(int k=1; k<=10000; k++) {
		int num=k,r,sum=0,temp=num,p,i;
		int count=0;
		while(temp!=0) {
			count++;
			temp=temp/10;
		}
		temp=num;
		while(temp!=0) {
			r=temp%10;
			p=1;
			for(i=1; i<=count; i++) {
				p=p*r;
			}
			sum=sum+p;
			temp=temp/10;
		}
		if(num==sum) {
			System.out.printf("%d is Armstrong\n",num);
		}

	}
}
 }